### ISSUE TITLE
Deploy 0.12.0?

### ISSUE THREAD

**Issue by abitrolly** [release-monitoring/anitya#561]
0.12.0 was released May 7 - https://github.com/release-monitoring/anitya/releases but still not deployed at https://release-monitoring.org/  Is there any problem that can be helped?

**Comment by Zlopez** [comment]
There is a ticket for this in [pagure.io](https://pagure.io/fedora-infrastructure/issue/6956).
I see it's created by you actually.

**Comment by abitrolly** [comment]
Yes, the ticket is still open after 3 months and I am not sure I want the update. :D

Docs on staging looks like a quick hack. Previous version was good that it fitted general design. If Sphinx is so hard to customize, maybe [MkDocs](https://www.mkdocs.org/) is a better option.

I like visual aesthetics of 0.11.0 version.

![image](https://user-images.githubusercontent.com/8781107/44296079-da660980-a2e9-11e8-8889-cdc72395b1cd.png)
![image](https://user-images.githubusercontent.com/8781107/44296097-20bb6880-a2ea-11e8-9fd0-83c4784e51a8.png)

Compared to 0.12.0.

![image](https://user-images.githubusercontent.com/8781107/44296088-01244000-a2ea-11e8-8826-c3854b2ab7b8.png)

Old docs.
![image](https://user-images.githubusercontent.com/8781107/44296113-5a8c6f00-a2ea-11e8-8e3f-48627304cdb9.png)

New docs.
![image](https://user-images.githubusercontent.com/8781107/44296116-6710c780-a2ea-11e8-89f9-547a95de1cf2.png)

**Comment by abitrolly** [comment]
As you may see Alabaster theme contains a lot of duplication, the navigation is too much and not enough at the same time, and it makes every project look just about the same.

**Comment by Zlopez** [comment]
I need to first release 0.13.0, which contains implementation of GitHub API. Than this needs to be deployed on staging and after this, we could roll this out on production. Hopefully this wouldn't take three months.

**Comment by yarons** [comment]
0.13.1 released so I'm guessing this one is closed?
